src/js/protocols/stm32usbdfu.js

```diff
@@ -298,6 +300,7 @@ STM32DFU_protocol.prototype.getFunctionalDescriptor = function (_interface, call
         'length':       255,
     }, function (result) {
         if (checkChromeRuntimeError()) {
+            GUI.log('Some kind of Chrome Runtime error happened');
```

Comment:
I trigger this error every time I flash.  Nothing bad seems to happen.  The debugger says:
```
Chrome API Error: Transfer stalled..
```
and the console log says
```
USB getFunctionalDescriptor failed! 4
```
despite this error, the flash always is successful